### PR TITLE
Corrupt Videos

### LINKED ISSUE #33: Corrupt Videos
Videos are being pull across to the pi3 before they are finished.

Solution: Create a folder called IntermediateVideos and move them over to Videos once completed

Tested and working

### PR DESCRIPTION
**Pull request by jw2476**
Fixes #33

### CODE REVIEW

File: app/remoteCam/video.py [hunk 1]
```diff
@@ -14,8 +14,8 @@
 
 if __name__ == "__main__":
 #    irled = led.sensor(3)  # Instantiate led class and assign the pin the BCM3
-
-    of = "/home/pi/Videos/"  # output folder
+    of = "/home/pi/IntermediateVideos/" # output folder
```
Comment (sebbacon) [inline app/remoteCam/video.py]: Avoid variable names like this. It saves a few keystrokes but makes reading the code much harder.

`working_folder` and `final_folder` would be much  better than `of` and `ff`.

File: app/remoteCam/video.py [hunk 2]
```diff
@@ -73,11 +73,11 @@
     d = output[:-9]
     d = d.replace("-", " ").replace("T", " ").replace(":", " ")
 
-    starttime = datetime.strptime(d, "%Y %m %d %H %M %S %z")
+    starttime = datetime.strptime(d, "%Y %m %d %H %M %S")
```
Comment (sebbacon) [inline app/remoteCam/video.py]: I see this is a "drive-by" bugfix which isn't related to the subject of the pull request.

Ideally this would be in a separate PR which relates to a separate bug report in a github issue.

Less ideally, but still better, is for it to be in a separate commit (perhaps with "drive-by bug fix" in the commit comment).   That git could explain what the bug fix is about (in the absence of a git issue about it).

To summarise, in an ideal world, each commit does a set of tightly-related things, and a PR is a set of commits related to a general topic.

That said I wouldn't waste time fixing that here, just something to bear in mind :)

Comment (jw2476) [reply app/remoteCam/video.py]: Thanks for pointing this out, that wasn't meant to change. I'll change it back now.

Comment (jw2476) [reply app/remoteCam/video.py]: Oh wait, I remember now. I put the %z in the wrong line, it got added to the correct line in master but it wasn't removed from the wrong one. This was fixed about a week ago when Dad was doing stuff so it wasn't pushed to git.

**Comment by jw2476** [comment]
@sebbacon lets get this merged in

File: app/remoteCam/video.py [hunk 3]
```diff
@@ -14,10 +14,10 @@
...
 if __name__ == "__main__":
 #    irled = led.sensor(3)  # Instantiate led class and assign the pin the BCM3
-    of = "/home/pi/IntermediateVideos/" # output folder
-    ff = "/home/pi/Videos/" # final folder
-    of1 = of + "1stPASS.mp4"
-    rectime = "10"  # record time of 10s
+    working_folder = "/home/pi/IntermediateVideos/" # output folder
+    final_folder = "/home/pi/Videos/" # final folder
+    output_file1 = working_folder + "1stPASS.mp4"
+    rectime = "10"  # record time working_folder 10s
```
Comment (sebbacon) [inline app/remoteCam/video.py]: I think you did a find-and-replace here and this should still read "record time of 10s"?!

File: app/remoteCam/video.py [hunk 4]
```diff
@@ -48,13 +48,13 @@
             "-t",
             rectime,
             "-y",
-            of1,
+            output_file1,
         ]
     )
     ffmpeg1.wait()
-  #  irled.off()  # Turn led off
+  #  irled.working_folderf()  # Turn led working_folderf
```
Comment (sebbacon) [inline app/remoteCam/video.py]: Ditto here

File: app/remoteCam/video.py [hunk 5]
```diff
@@ -48,13 +48,13 @@
...
 
-    # ffprobe to extract recording time and date and turn into offset seconds
+    # ffprobe to extract recording time and date and turn into working_folderfset seconds
```
Comment (sebbacon) [inline app/remoteCam/video.py]: Ditto here

File: app/remoteCam/video.py [hunk 6]
```diff
@@ -75,24 +75,24 @@
 
     starttime = datetime.strptime(d, "%Y %m %d %H %M %S")
     local_time = starttime.replace(tzinfo=pytz.utc).astimezone(local_timezone)
-    offset = local_time.timestamp()
-    logger.debug("Computed date offset %s", offset)
+    working_folderfset = local_time.timestamp()
```
Comment (sebbacon) [inline app/remoteCam/video.py]: Ditto here

File: app/remoteCam/video.py [hunk 7]
```diff
@@ -75,24 +75,24 @@
...
+    logger.debug("Computed date working_folderfset %s", working_folderfset)
```
Comment (sebbacon) [inline app/remoteCam/video.py]: etc...! won't mark all the rest

File: app/remoteCam/video.py (hunk 3 above)
Comment (jw2476) [reply app/remoteCam/video.py]: Yep will fix

**Comment by jw2476** [comment]
I'm am going to redo this branch as well, do it manually this time
